src/services/auth-services.ts

```diff
@@ -114,7 +114,7 @@ export default class AuthServices {
 
     const authApiUrl = sessionStorage.getItem('AUTH_API_URL')
     const url = `${authApiUrl}entities/${businessId}`
-    const data = { businessId, folioNumber }
+    const data = { businessIdentifier: businessId, folioNumber: folioNumber ?? '' }
```

Comment:
I think `??` isn't the right operator. Do you mean '||' (as in, if folioNumber is falsy then fall back to empty string)?

Also, as this is an object, sometimes it's nice to put the properties on separate lines. Not really needed here unless you want to add a comment beside the second property to say what you're doing?

Also, do you want to fall back to an empty string, null, or undefined (which takes the property out of the resulting JSON object)?

Reply:
`??` checks to see if undefined or null - They probably ported this over from C#, my intent was to avoid a null or undefined folioNumber, it causes the API call to fail

`||` = truthy  0 || "not found" -> "not found"

I think either operator could be used here, but i'll change it to ||